lib/Test/Nginx/Util.pm

```diff
@@ -174,6 +174,39 @@ sub gen_rand_str {
     return $s;
 }
 
+sub gen_rand_port (;$$) {
```

Comment:
Here you assume the `tries` argument can be optional. But you never initialize it when it is not defined?

Reply:
I missed it. It's been added.